src/dealloc.F90

```diff
@@ -0,0 +1,225 @@
+subroutine deallocate_all() 
+use mod_streams
+
+if (allocated(wv_gpu)) deallocate(wv_gpu)
+if (allocated(w_order)) deallocate(w_order)
+if (allocated(wallpfield_gpu)) deallocate(wallpfield_gpu)
+if (allocated(slicexy_gpu)) deallocate(slicexy_gpu)
+if (allocated(vf_df_old)) deallocate(vf_df_old)
```

Comment:
deallocate CPU-side arrays if they exist. `deallocate_all` is bound out to python